scalalib/src/scoverage/ScoverageModule.scala

```diff
@@ -0,0 +1,48 @@
+package mill.lib.scoverage
```

Comment:
I think, it would be better to add under `mill.contrib.scoverage`.

Documentation then goes to the `Contrib Module` pages.

Reply:
Re: package location
d41d1d51faa3259c1af5989dfe5a12c44e2bdb85